lib/load-config.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const requireEsm = require('esm')(module);
+const path = require('path');
+const fs = require('fs');
+const pkgConf = require('pkg-conf');
+
+const loadConfigFile = projectDir => {
+	const loaded = requireEsm(path.resolve(projectDir, 'ava.config.js'));
+	const config = (loaded.__esModule) ? loaded.default : loaded;
```

Comment:
I want to play around with the [`esm` options](https://www.npmjs.com/package/esm#options). I *think* we can make it always treat these files as ESM, so we don't have to support the `module.exports` workarounds etc. Would be neat to start with that. What do you think @sindresorhus?

Reply:
👍